src/main/kotlin/minesweeper/entity/MineCounting.kt

```diff
@@ -0,0 +1,28 @@
+package minesweeper.entity
+
+class MineCounting(private val minedMap: List<List<MapElement>>, private val mapInfo: MapInformation) {
+    val dx = listOf<Int>(0, 1, 0, -1)
+    val dy = listOf<Int>(-1, 0, 1, 0)
+
+    fun searchMap(): List<List<MapElement>> {
+        return minedMap.mapIndexed { rowIndex: Int, row: List<MapElement> ->
+            List(row.size) { colIndex -> searchAllDirection(rowIndex, colIndex) }
+        }
+    }
+
+    fun searchAllDirection(curX: Int, curY: Int): MapElement {
+        var countOfMines = 0
+        if (minedMap[curX][curY] == MapElement.MINE) return MapElement.MINE
```

Comment:
하나의 라인으로 표시해줄 수 있더라도, 중괄호로 묶어주면 어떨까요?

```suggestion
        if (minedMap[curX][curY] == MapElement.MINE) {
            return MapElement.MINE
        }
```